docs/NEW_GUIDE_DRAFT.en.md

```diff
@@ -0,0 +1,97 @@
+`NEAR CLI` is using `extensions` to saticfy all groups of users.
+    - `Core NEAR CLI` commands should be usefull for all groups of users.
+    - `Core NEAR CLI` is a single binary, there is no extensions that are installed `by default`.
+    - Each extension is a separate binary that can be executed from `NEAR CLI`.
+    - Extensions are used by a particular group or several groups of users.
+    - Extensions are not composable (you should not create extensions for extensions).
+
+`NEAR CLI` is built for:
+    - NEAR `dApp developers`, who build smart-contracts, UIs, and tooling on NEAR.
+    - `Tech-savvy` people automating their routines.
+    - `Validators`
+    - ...
+
+`UX principles`
+- All altering actions should have a confirmation step with an option to skip confirmation with an explicit command line parameter (e.g. send at the end of the command)
+- All direct children commands of a single parent command should be aligned (either represent an action or a resource, but never a mix of those on the same hierarchy level): contract -> state (resource) and contract -> deploy (action) are not aligned, so it should be either contract -> get-state + contract -> deploy or contract -> state -> view + contract -> code -> deploy
+- Interactive mode should look like: "command - description". It will help people to learn the commands.
+
+
+`Core NEAR CLI` commands:
+import-account
+    - from-near-wallet
+    - from-seed-phrase
+    - from-ledger
+    - from-private-key
+create-account
+    - implicit
+    - subaccount
+manage-account <accoundId>
+    - get-state
+    - manage-keys
+        - view
+        - add
+        - delete
+    - make-transfer
```

Comment:
would that encompass sending NEAR or would sending NEAR remain in the `near send` command

Reply:
@frol FT and NFT sound like yet another extension. `transfer-tokens` is ok. I'm more worried about the structure of commands here.
@BenKurrek There is no `near send` in the current design. It was at the top level when this PR was created, but now it somewhere in `near` -> `account` -> `mange` -> `state` -> `transfer-tokens` or `near` -> `manage-account` -> `transfer-tokes`.

Reply:
I tend to agree that from the design perspective FT and NFT should be in dedicated extensions, but it might (yet to be confirmed) be a common use-case; yet, even in that case, I would vote for a dedicated extension and just make sure that installing new extensions is as smooth as possible.